hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -24,7 +24,7 @@
   ],
 
   param_list: [
-    { name: "NumBanks",
+    { name: "NBanks",
```

Comment:
soo...this parameter is conflicting with the one i'm using in the normal flash_ctrl_pkg.sv
Once I template, I don't think this matters...but let me know what you think.

Reply:
do you mean you are going to align them later on such that there is only one source definition of this parameter?

Reply:
no i'm just going to remove this one altogether. When this is templated, the number for multireg count will be filled in by the template.

Reply:
actually it's probably cleaner for me to just stick all of my pkg's into the localparam list for reg_pkg... yeah so it'll probably end up being 1 like you said, but in the reg_pkg instead of _pkg

Reply:
ok, makes sense. I think I did that as well for the alert handler, where the top-level config gets passed through the templated hjson, and then I just grab these params directly from the `reg_pkg` in RTL.